.circleci/config.yml

```diff
@@ -166,6 +166,21 @@ jobs:
       #    is triggered automatically via GitHub webhook on https://readthedocs.org/projects/git-machete/builds/,
       # 3) every time a https://github.com/r-ryantm bot runs on NixOS/nixpkgs repository,
       #    Nix git-machete package is updated with the fresh version/package hash according to the data from PyPI.
+
+  "Publish image to Docker Hub":
+    working_directory: /git-machete
+    docker:
+      - image: docker:17.05.0-ce-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - deploy:
+          name: Publish application to docker hub
+          command: |
+            echo $DOCKER_PASSWORD | docker login -u gitmachete --password-stdin
```

Comment:
But here `$DOCKER_LOGIN` is desired!
Since for logging in, we use some **technical user** (whose name is stored in `DOCKER_LOGIN` variable), but we push to images under `gitmachete` organization (https://hub.docker.com/u/gitmachete)